provider/go.mod

```diff
@@ -4,13 +4,13 @@ go 1.19
 
 replace (
 	github.com/hashicorp/go-getter v1.5.0 => github.com/hashicorp/go-getter v1.4.0
-	github.com/hashicorp/terraform-plugin-sdk/v2 => github.com/pulumi/terraform-plugin-sdk/v2 v2.0.0-20221122203342-430f685de305
+	github.com/hashicorp/terraform-plugin-sdk/v2 => github.com/pulumi/terraform-plugin-sdk/v2 v2.0.0-20230327102345-3fa930f86570
```

Comment:
It's this one? https://github.com/pulumi/terraform-plugin-sdk/tree/upstream-v2.26.1 ? 

Curious.

Reply:
Do you mean the cause of the breaking changes?

Reply:
No, I was curious why the hash of the github.com/pulumi/terraform-plugin-sdk fork is changing and is it in fact an upgrade; looks like it is, looks like Ian pushed a more recent branch there. So all good, just not something I noticed before.